site/docs/proc_apache-camel.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Knative Camel Operator :: Knative Eventing on OpenShift Container Platform</title>
+    <link rel="canonical" href="https://openshift-knative.github.io/docs/docs/proc_apache-camel.html">
```

Comment:
If you've changed the file path, I would check and see if this link is still valid. It might be less hassle in the long wrong to modify the .adoc or YAML file, instead of making changes directly to the HTML file.